### DISCUSSION TITLE
使用的疑问

### DISCUSSION THREAD

**Discussion by yukixsp** [amll-dev/applemusic-like-lyrics#291] [Q&A]
现在我点击左下角的专辑界面呼出歌词时，会先跳出官方的歌词界面，关掉之后才会出现插件设置的类苹果歌词。请问是我的设置有问题或是使用有什么问题吗？

**Comment by teea418** [comment]
同问

**Comment by Linho1219** [comment]
AMLL bncm 插件已停止维护，最终版本存档于 <https://github.com/amll-dev/applemusic-like-lyrics-proto>。

现关闭此讨论，敬请谅解。
